sktime/forecasting/compose/_fallback.py

```diff
@@ -96,14 +114,17 @@ class is useful in scenarios where the reliability of individual forecasting mod
     # this must be an iterable of (name: str, estimator, ...) tuples for the default
     _steps_fitted_attr = "forecasters_"
 
-    def __init__(self, forecasters, verbose=False):
+    def __init__(self, forecasters, verbose=False, nan_predict_policy="ignore"):
         super().__init__()
 
         self.forecasters = forecasters
         self.current_forecaster_ = None
         self.current_name_ = None
         self.verbose = verbose
 
+        _check_nan_policy_option(nan_predict_policy)
+        self.nan_predict_policy = nan_predict_policy
```

Comment:
slightly more idiomatic (similar ot sklearn): have the checker return the input, then do `self.palim = check_palim(palim)`